packages/vscode-extension/src/plugins/expo-dev-plugins/ExpoDevPluginWebviewProvider.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  CancellationToken,
+  ExtensionContext,
+  Uri,
+  WebviewView,
+  WebviewViewProvider,
+  WebviewViewResolveContext,
+} from "vscode";
+import { IDE } from "../../project/ide";
+import { ExpoDevPluginToolName } from "./expo-dev-plugins";
+import { Logger } from "../../Logger";
+
+function generateWebviewContent(pluginName: ExpoDevPluginToolName, metroPort: number): string {
+  const iframeURL = `http://localhost:${metroPort}/_expo/plugins/${pluginName}`;
+  return /*html*/ `
+    <!DOCTYPE html>
+    <html style="height: 100%;">
+      <head/>
+      <body style="height: 100%;">
```

Comment:
```suggestion
      <body style="height: 100%; overflow:hidden">
```

this will remove the scroll bar next to the panel.